virttest/backends/v2v/cfg/convert_source.cfg

```diff
@@ -37,4 +37,10 @@ variants:
                 vpx_hostname = VPX_70_HOSTNAME_V2V_EXAMPLE
                 vpx_dc = VPX_70_DC_V2V_EXAMPLE
                 esx_hostname = ESX_70_HOSTNAME_V2V_EXAMPLE
-                vddk_libdir_src = MOUNT_SRC_VDDK70_LIB_DIR_V2V_EXAMPLE
\ No newline at end of file
+                vddk_libdir_src = MOUNT_SRC_VDDK70_LIB_DIR_V2V_EXAMPLE
+            - esx_80:
+                datastore = VPX_80_DATASTORE_V2V_EXAMPLE
+                vpx_hostname = VPX_80_HOSTNAME_V2V_EXAMPLE
+                vpx_dc = VPX_80_DC_V2V_EXAMPLE
+                esx_hostname = ESX_80_HOSTNAME_V2V_EXAMPLE
+                vddk_libdir_src = MOUNT_SRC_VDDK70_LIB_DIR_V2V_EXAMPLE
```

Comment:
what vddk version is it? vddk80 or 70?

Reply:
Just ask mxie to copy vddk8.0.0 to NFS. Will update to 8.0 then.